src/ProgressMeter.jl

```diff
@@ -324,23 +324,23 @@ end
 
 # update progress display
 """
-`next!(prog, [color])` reports that one unit of progress has been
+`next!(prog, [color], step = 1)` reports that step units of progress has been
```

Comment:
```suggestion
`next!(prog, [color], step = 1)` reports that `step` units of progress have been
```

Reply:
I also changed the definition of the next!() without color argument. As Symbol != Int I do not see a possible conflict. If you like the keyword approach more I would also be perfectly happy with that.

Reply:
Right, there isn't a possible conflict, but then someone might implement a positional version where `step` comes second and it supplies the default for `color`. So that's an extra method, one that introduces two places we have to have the default `color` value hard-coded.